CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
-# Synergy Design System Contribution Guidelines
```

Comment:
I'd rather let the wording as it is. Synergy is a design system, thats a fact. Leaving it away would just add confusion. And we explain it to the world here and not only to Sick internals.

Reply:
The name is Synergy, and not Synergy Design System. 
In all communication we strife to avoid the "design system" part, as discussed and agreed with CorpCom, David, Timothy and us.
I'll re-add it where it males sense in the content, but I want to reduce the name to Synergy in the headlines.
If needed we can discuss it with Carine next week.